tests/InterNations/Tests/Integration/BundleIntegrationTest.php

```diff
@@ -1,11 +1,10 @@
 <?php
```

Comment:
@lstrojny how bad an idea is it that I only moved BundleIntegrationTest to ./tests?

Reply:
I tried to do that first! But i was gettig these kind of errors for all the files which are under /app and /fixture.

14) InterNations\Bundle\TypeJailBundle\Tests\Integration\BundleIntegrationTest::testRenderAggregateTemplate
Error: Class 'InterNations\Bundle\TypeJailBundle\Tests\Integration\app\AppKernel' not found

/Users/malvikachauhan/Projects/type-jail-bundle/tests/InterNations/tests/Integration/BundleIntegrationTest.php:99
/Users/malvikachauhan/Projects/type-jail-bundle/tests/InterNations/tests/Integration/BundleIntegrationTest.php:86
phpvfscomposer:///Users/malvikachauhan/Projects/type-jail-bundle/vendor/phpunit/phpunit/phpunit:97

Reply:
Did you configure a dev auto loader?

Reply:
I configured testsuite path. Where else i need configuration?
<testsuites>
        <testsuite name="internations/type-jail-bundle">
            <directory>./tests</directory>
        </testsuite>
    </testsuites>

Reply:
Check this out https://getcomposer.org/doc/04-schema.md#autoload-dev

Reply:
I will do, thank you!

Reply:
Done! And it's working now. Thank you very much. I guess now i am ready to merge all the 4 PRs, and bump the minor version for this project.

Reply:
Congrats and yes, looks good